src/product-card-buy.scss

```diff
@@ -0,0 +1,30 @@
+.product-card__buy {
+  cursor: pointer;
+  box-sizing: border-box;
+  display: flex;
+  background-color: $buyColor;
+  margin: $productCardMargin;
```

Comment:
you give margins this way almost everywhere for your card items. better let's add padding for whole card, and for items just margin-top or bottom